### PR TITLE
Ns fix alignment

### PR DESCRIPTION
**Pull request by valeriabarra**
This just fix a wrong alignment that was introduced in the NS PR #152

### CODE REVIEW

File: tests/t306-basis.c [hunk 1]
```diff
@@ -2,6 +2,7 @@
 /// Test QR Factorization
 /// \test Test QR Factorization
 #include <ceed.h>
+#include <ceed-impl.h>
```
Comment (jeremylt) [inline tests/t306-basis.c]: I don't think this is needed. Otherwise I'm happy to squash+merge this.

Comment (valeriabarra) [reply tests/t306-basis.c]: I'm not even sure when that was introduced.. I'll remove it

File: examples/petsc/Makefile [hunk 2]
```diff
@@ -14,6 +14,9 @@ LDLIBS = $(call pkgconf, --libs-only-l $(PETSc.pc) $(ceed.pc)) -lm
 bp.c := $(sort $(wildcard bp*.c))
 bp := $(bp.c:%.c=%)
 
+ns.c := ns.c
```
Comment (jeremylt) [inline examples/petsc/Makefile]: Sorry to keep missing things - The changes in this file are a holdover from when the Navier-Stokes example was in the PETSc folder, right?

Comment (valeriabarra) [reply examples/petsc/Makefile]: Thanks for noticing that. Yeah, that was very old

**Pull request merged**